components/EmotionList.js

```diff
@@ -30,7 +36,7 @@ export default function EmotionList() {
       <h1>The seven basic emotions</h1>
       {emotionData.map(({ slug, name, color }) => (
         <StyledLi key={slug} $color={color}>
-          {name}
+          <StyledLink href={`emotions/${slug}`}>{name}</StyledLink>
```

Comment:
I think we want this to be an absolute url, rather than a relative one. 

> The path with reference to root directory is called absolute. The path with reference to current directory is called relative.

```suggestion
          <StyledLink href={`/emotions/${slug}`}>{name}</StyledLink>
```